src/ceph_mon.cc

```diff
@@ -710,6 +710,12 @@ int main(int argc, const char **argv)
     prefork.exit(1);
   }
 
+  Messenger *mgr_msgr = Messenger::create(g_ceph_context, public_msgr_type,
```

Comment:
we have to have a separate messenger because it's a different policy.  one is a lossy client and one a lossy server.  note that the only "overhead" from that is a separate dispatch thread.  i didn't use a unique_ptr because none of the other msgrs do and wanted to keep it symmetric